tlatools/org.lamport.tlatools/src/tla2tex/Misc.java

```diff
@@ -364,7 +363,20 @@ public static boolean IsLetter(char c)
                  || ( ('A' <= c ) && (c <= 'Z') )
                  || ( c == '_' ) ;} ;
 
-                 
+    /**
+     * IsLetter function implemented for Unicode codepoints.
+     * 
+     * @param c The codepoint to check.
+     * @return Whether the given codepoint is a letter.
+     */
+    public static boolean IsLetter(int c) {
+        if (c >= 128) {
```

Comment:
Do codepoints use all 32 bits of the `int`? (I actually don't know the answer myself.)

If so, then very large ones will overflow, so we might need to revise this check to `c < 0 || c >= 128` to handle that case. (Same for the other `Is*` methods below.)

Reply:
Good catch!